### DISCUSSION TITLE
Study Design

### DISCUSSION THREAD

**Discussion by msevestre** [Open-Systems-Pharmacology/OSPSuite.Core#764] [Ideas]
Idea: A list of conditions and modifications that will be applied to a population during before simulating and individual

For example
If BW >10kg and BW <30kg and Gender = Male, set application|DrugMass to 25mg
If BW >= 30kg set application|DrugMass to 40mg

To support the study design concept in the reporting engine, I propose to implement the following in Core

Study Design with a list of StudyEffect
One Effect has multiple Conditions (filtering) and Multiple Modifications (what will be changed)

Nomenclature:
I like the term of effect. This will be an internal term and won't be used

As far as input vs output I see

Conditions or Filters
Modifications or Start Values. 

**Conndition:** 
Path (e.g. Organism|BodyWeight)
Value : 20
Unit: kg
Operator (> >=, <=, =, != etc..)

**Modification:**
Path (e.g. Application|App1|DrugMass)
Value: 40
Unit: mg

This could become a building block in a next iteration

Such a study design could be loaded from a csv file 
see https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/issues/24 for specs

It is not required to have all individuals updated by a condition
Some conditions may cover multiple individuals
A check will be performed to ensure that the modification are modifying parameters that do exist in the simulation

Please feedback @Yuri05

**Comment by Yuri05** [comment]
Like it! :)
